app/pb_kits/playbook/pb_selectable_icon/selectable_icon.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+module Playbook
```

Comment:
Shouldn't we move to the new model since this ins't merged yet?

Reply:
I believe Brave and The Bold is handling those migrations independently.